pkg/anonymiser/anonymiser.go

```diff
@@ -17,75 +17,55 @@ const literalPrefix = "literal:"
 // anonymiser anonymises MySQL tables
 type anonymiser struct {
 	reader.Reader
+	tables config.Tables
 }
 
 // NewAnonymiser returns an initialised instance of MySQLAnonymiser
-func NewAnonymiser(source reader.Reader) reader.Reader {
-	return &anonymiser{source}
+func NewAnonymiser(source reader.Reader, tables config.Tables) reader.Reader {
+	return &anonymiser{source, tables}
 }
 
-func (a *anonymiser) ReadTable(table string, rowChan chan<- *database.Row) error {
-	logger := log.WithField("table", table)
+func (a *anonymiser) ReadTable(tableName string, rowChan chan<- *database.Row) error {
+	logger := log.WithField("table", tableName)
+
 	logger.Info("Loading anonymiser config")
-	// Find all columns that need to be anonimized
-	columnFakers, err := a.fetchColumnToAnonimise(table)
+	table, err := a.tables.FindByName(tableName)
 	if err != nil {
-		close(rowChan)
-		logger.WithError(err).Error("Failed to load anonymiser config")
-		return err
+		logger.WithError(err).Warn("the table is not configured to be anonymised")
+		return nil
```

Comment:
This should still trigger the dump for the table right? same as line https://github.com/hellofresh/klepto/pull/40/files#diff-193869b0771795a3496d4ec90d2e096aR40